packages/features/settings/TimezoneChangeDialog.tsx

```diff
@@ -104,9 +82,56 @@ export function useOpenTimezoneDialog() {
 
 export default function TimezoneChangeDialog() {
   const { open, setOpen, browserTimezone } = useOpenTimezoneDialog();
+  const { t } = useLocale();
+  const utils = trpc.useUtils();
+
+  const [selectAvailability, setSelectAvailability] = useState(false);
+  const [ids, setIds] = useState<number[]>([]);
+  const { data: availability, isPending } = trpc.viewer.availability.list.useQuery();
+  const formattedCurrentTz = browserTimezone.replace("_", " ");
+
+  const data = availability?.schedules?.map((a) => ({
+    id: a.id,
+    title: a.name,
+    default: a.isDefault,
+  }));
+
+  const mutation = trpc.viewer.updateProfile.useMutation({
+    onSuccess: async () => {
+      showToast(t("updated_timezone_to", { formattedCurrentTz }), "success");
+      await utils.viewer.me.invalidate();
+      setSelectAvailability(false);
+    },
+    onError: () => {
+      showToast(t("couldnt_update_timezone"), "error");
+    },
+  });
+
   return (
     <Dialog open={open} onOpenChange={setOpen}>
-      <TimezoneChangeDialogContent browserTimezone={browserTimezone} onAction={() => setOpen(false)} />
+      <TimezoneChangeDialogContent
+        browserTimezone={browserTimezone}
+        onAction={() => {
+          setOpen(false);
+          setSelectAvailability(true);
+        }}
+      />
+      <BulkEditDefaultModal
+        title={t("default_timezone_bulk_title")}
+        description={t("default_timezone_bulk_description")}
```

Comment:
I think we need to change the copy in the description 

> Update the locations for the selected event types

But we're actually updating availability

Reply:
Yeah missed that my bad, fixed it.